pkg/gardenlet/controller/shoot/shoot_control_reconcile.go

```diff
@@ -290,6 +290,16 @@ func (c *Controller) runReconcileShootFlow(ctx context.Context, o *operation.Ope
 			Fn:           flow.TaskFn(botanist.DeployKubeScheduler).RetryUntilTimeout(defaultInterval, defaultTimeout),
 			Dependencies: flow.NewTaskIDs(deploySecrets, waitUntilKubeAPIServerIsReady),
 		})
+		deployKonnectivityServer = g.Add(flow.Task{
+			Name:         "Deploying konnectivity-server",
+			Fn:           flow.TaskFn(botanist.DeployKonnectivityServer).RetryUntilTimeout(defaultInterval, defaultTimeout),
```

Comment:
```suggestion
			Fn:           flow.TaskFn(botanist.DeployKonnectivityServer).DoIf(o.Shoot.KonnectivityTunnelEnabled && useSNI).RetryUntilTimeout(defaultInterval, defaultTimeout),
```

Otherwise fails with

> "Deploying konnectivity-server\" failed: retry failed with context deadline exceeded, last error: missing client-ca secret information

Reply:
not resolved?

Reply:
Not needed as now `DefaultKonnectivityServer` handles this.